src/catkin_pkg/templates/package.xml.in

```diff
@@ -26,22 +26,128 @@
   <!-- <author email="jane.doe@@example.com">Jane Doe</author> -->
 @authors
 
-  <!-- The *_depend tags are used to specify dependencies -->
+  <!-- Packages can have seven types of dependencies -->
   <!-- Dependencies can be catkin packages or system dependencies -->
-  <!-- Examples: -->
-  <!-- Use build_depend for packages you need at compile time: -->
-  <!--   <build_depend>message_generation</build_depend> -->
-  <!-- Use buildtool_depend for build tool packages: -->
-  <!--   <buildtool_depend>catkin</buildtool_depend> -->
-  <!-- Use run_depend for packages you need at runtime: -->
-  <!--   <run_depend>message_runtime</run_depend> -->
-  <!-- Use test_depend for packages you need only for testing: -->
-  <!--   <test_depend>gtest</test_depend> -->
-@dependencies
+
+<!-- Build: -->
+
+  <!-- <buildtool_depend> -->
+  <!-- Specify build system tools which this package needs to build itself.
+       Typically the only build tool needed is catkin.
+	   In a cross-compilation scenario build tool dependencies are for the 
+	   architecture on which the compilation is performed. -->
+  <!-- <buildtool_depend>catkin</buildtool_depend> -->
+@buildtool_depends
+
+  <!-- <depend> -->
+  <!-- Specifies that a dependency is a build, export, and execution dependency.
+       This is the most commonly used dependency tag -->
```

Comment:
> This is the most commonly used dependency tag

This should *not* be the most commonly one being used. This should only be used if a dependency is truely both (build and run dependency). Or if the user really doesn't care about the dependencies of generated binary packages.